File: practices/fizzbuzz.py
# MM 6th FizzBuzz

for x in range(1,51):
    if x % 3 == 0:
        if x % 5 == 0:
            print("FizzBuzz")
        else:
            print("Fizz")
    elif x % 5 == 0:
        print("Buzz")
    else:
        print(x)

File: c_programming/practices/fizzbuzz.c
// MM CSP 6th Fizz Buzz
#include <stdio.h>

int main(void){

    for (int num = 1; num < 51; num++){
        if (num % 3 == 0 && num % 5 == 0){
        printf("FizzBuzz\n");
        }else if (num % 3 == 0){
            printf("Fizz\n");
        }else if (num % 5 == 0){
            printf("Buzz\n");
        }else{
            printf("%d\n", num);
        }
    }

    return 0;
}